.github/workflows/build.yml

```diff
@@ -5,9 +5,13 @@ on:
     types:
       - opened
       - synchronize
+    paths-ignore:
+      - docs/**
   push:
     branches:
       - main
+    paths-ignore:
+      - docs/**
```

Comment:
```suggestion
```

For this PR, it's sufficient only to add paths-ignore for the pr target. The push target is used for release builds and there is some reliance on the documentation site on releases being built/created.